helpdesk_mgmt_timesheet/__manifest__.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+#    Copyright (C) 2020 Aresoltec Canarias <www.aresoltec.com>
+#    Copyright (C) 2020 Punt Sistemes <www.puntsistemes.es.es>
+#    Copyright (C) 2020 SDi Soluciones Digitales <www.sdi.es>
+#
+#    This program is free software: you can redistribute it and/or modify
```

Comment:
These bloated headers are obsolete since a lot. Please use the reduced ones:

https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L1-L2

Reply:
Fixed!